File: src/ui/app/root/application/models/Assessment/AssessmentSectionMap.ts
export interface IAssessmentSectionMap {
    assessmentSectionMapId: string;
    assessmentSchemeMasterId: string;
    sectionCourseLinkId: string;
    statusId: number;
}

export interface IVWAsssessmentSectionMap {
    assessmentSectionMapId: string;
    assessmentSchemeMasterId: string;
    assessmentMaster: string;
    sectionId: string;
    sessionId : string;
    campusId : string;
    campusName : string;
    programId : string ;
    programName : string;
    classId : string;
    className : string;
    campusProgramId : string;
    sectionCourseLinkId : string;
    sectionName : string ;
    levelId : string ;
    levelName : string;
    statusId: number;
}

export interface IAssessmentSchemeMaster1 {
        assessmentSchemeMasterId : string;
        fullName : string;	
        gradingMasterId : string;
        failMasterId : string;
        totalWeightage : string;
		statusId : number;
}

export interface IAssessmentProgramLevel {
    programId : string;
    programName : string;	
    levelId : string;
    programDetailId : string;
    campusProgramId : string;
    statusId : number;
}

// export interface CheckCount {
//     response : number;
// }

export interface IAssessmentClassLevel {
    programId : string;
    programName : string;	
    levelId : string;
    programDetailId : string;
    campusProgramId : string;
    classId : string;
    className : string;
    statusId : number;
}

File: src/data/Model/Cms360/Assessment/AssessmentSectionMap.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
    [Table("AssessmentSectionMap", Schema = "Assessment")]
    public partial class AssessmentSectionMap
    {
		[Key]
		[Required]
        public Guid AssessmentSectionMapId { get; set; }

		public Guid AssessmentSchemeMasterId { get; set; }

		public Guid SectionCourseLinkId { get; set; }

		public int StatusId { get; set; }

    }

	[Table("VWAssessmentSectionMap", Schema = "Assessment")]
    public partial class VWAssessmentSectionMap
    {
		[Key]
		[Required]
        public Guid AssessmentSectionMapId { get; set; }
		public Guid AssessmentSchemeMasterId { get; set; }
        public string AssessmentMaster { get; set; }
		public Guid SectionCourseLinkId { get; set; }
		public Guid SectionId { get; set; }
		public Guid ProgramDetailId { get; set; }
		public string SectionName { get; set; }
		public Guid CampusId { get; set; }
		public string CampusName { get; set; }	
		public Guid ProgramId { get; set; }
		public string ProgramName { get; set; }		
		public Guid ClassId { get; set; }
		public string ClassName { get; set; }
		public Guid LevelId { get; set; }
		public string LevelName { get; set; }
		public Guid SessionId { get; set; }
		public Guid CampusProgramId { get; set; }
		public int StatusId { get; set; }

    }

	 public partial class AssessmentLevelProgram
    {
		[Key]
		[Required]
        public Guid ProgramId { get; set; }

		public string ProgramName { get; set; }

		public Guid LevelId { get; set; }

		public Guid ProgramDetailId { get; set; }

		public Guid CampusProgramId { get; set; }

		public int StatusId { get; set; }

    }

}